python/lsst/ts/cbp/csc.py

```diff
@@ -83,14 +82,32 @@ async def do_move(self, data):
         """
         self.log.debug("Begin move")
         self.assert_enabled("move")
-        await asyncio.join(
-            [
-                self.model.move_elevation(data.elevation),
-                self.model.move_azimuth(data.azimuth),
-            ]
+        await asyncio.gather(
+            self.model.move_elevation(data.elevation),
+            self.model.move_azimuth(data.azimuth),
+        )
```

Comment:
Does this only move in elevation and azimuth?

If so, please refactor the code below so that it can also be called by the methods that trigger motion of the other actuators.

Even if it moves all axes, please consider having a generic "report_state" method that you can call (preferably from a callback function in the Model whenever state changes).